initfiles/componentfiles/thor/start_backupnode.in

```diff
@@ -71,7 +71,7 @@ if [ ! -z ${THORPRIMARY} ]; then
 else
     groupName=${THORNAME}
 fi
-daliadmin $DALISERVER dfsgroup ${groupName} > $INSTANCE_DIR/slaves
+daliadmin $DALISERVER dfsgroup ${groupName} $INSTANCE_DIR/backupnode.slaves
```

Comment:
Only thing I see is changing this to server=$DALISERVER like we did in other daliadmin calls throughout the init scripts.  Other than that it looks great.  @jakesmith